frontend/fingertips-frontend/lib/chartHelpers/pointFormatterHelper.test.ts

```diff
@@ -0,0 +1,37 @@
+import { pointFormatterHelper } from './pointFormatterHelper';
+
+const mockPoint = {
+  graphic: {
+    symbolName: '',
+    color: 'null',
+  },
+  y: 1,
+  series: {
+    name: '',
+  },
+};
+
+describe('pointFormatterHelper', () => {
+  it('should return a string with the default symbol code for circle when symbolName is empty string', () => {
+    const actual = pointFormatterHelper(mockPoint);
+    expect(actual).toContain('\u25CF');
+  });
+
+  it('should return a string with the correct symbol code', () => {
+    const symbolNames = [
+      'circle',
+      'square',
+      'diamond',
+      'triangle',
+      'triangle-down',
+    ];
+
+    const symbolCodes = ['\u25CF', '\u25a0', '\u25c6', '\u25b2', '\u25bC'];
+
+    symbolNames.forEach((name, i) => {
```

Comment:
Nice! but you can do something like this instead which I think will be more readable https://robertmarshall.dev/blog/how-to-loop-in-jest-unit-tests/